api/DOMPoint.json

```
@@ -98,6 +98,106 @@
           }
         }
       },
+      "fromPoint": {
+        "__compat": {
+          "description": "<code>DOMPoint()</code> constructor",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/DOMPoint/DOMPoint/fromPoint",
+          "spec_url": "https://drafts.fxtf.org/geometry/#dom-dompoint-frompoint",
+          "support": {
+            "chrome": {
+              "version_added": "61"
+            },
+            "chrome_android": {
+              "version_added": "61"
+            },
+            "edge": {
+              "version_added": "79"
+            },
+            "firefox": {
+              "version_added": "31"
+            },
+            "firefox_android": {
+              "version_added": "31"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "48"
+            },
+            "opera_android": {
+              "version_added": "45"
+            },
+            "safari": {
+              "version_added": "10.1"
+            },
+            "safari_ios": {
+              "version_added": "10.3"
+            },
+            "samsunginternet_android": {
+              "version_added": "8.0"
+            },
+            "webview_android": {
+              "version_added": "61"
+            }
+          },
+          "status": {
+            "experimental": true,
```

Mark all features in this file as non-experimental.

```suggestion
            "experimental": false,
```

---

Fixed in next commit.